Dockerfile

```diff
@@ -86,15 +85,17 @@ RUN systemctl set-default multi-user.target \
 		graphical.target \
 	&& systemctl disable ssh.service
 
-COPY src/confd.service src/balena-root-ca.service src/balena-host-envvars.service /etc/systemd/system/
-COPY src/configure-balena-root-ca.sh src/configure-balena-host-envvars.sh /usr/sbin/
+COPY src/confd.* src/certs-watch.* /etc/systemd/system/
```

Comment:
The only thing `confd.*` matches is `confd.service` which is the same as was there before - is there a missing file or something?

Reply:
@Page- it's just the service, I've made it explicit now.